enclave-tls/src/enclave_quotes/sgx-ecdsa-qve/main.c

```diff
@@ -22,7 +22,7 @@ extern enclave_quote_err_t sgx_ecdsa_cleanup(enclave_quote_ctx_t *ctx);
 
 static enclave_quote_opts_t sgx_ecdsa_qve_opts = {
 	.api_version = ENCLAVE_QUOTE_API_VERSION_DEFAULT,
-	.flags = ENCLAVE_QUOTE_FLAGS_DEFAULT,
+	.flags = ENCLAVE_QUOTE_OPTS_FLAGS_SGX_ENCLAVE,
 	.type = "sgx_ecdsa_qve",
```

Comment:
The `.flags` in `sgx-ecdsa/main.c` also need to change to `ENCLAVE_QUOTE_OPTS_FLAGS_SGX_ENCLAVE`

Reply:
@haosanzi Essentially, the attester portion of sgx-ecdsa instance requires ENCLAVE_QUOTE_OPTS_FLAGS_SGX_ENCLAVE but the verifier portion doesn't. So we cannot simply mark sgx-ecdsa instance with ENCLAVE_QUOTE_OPTS_FLAGS_SGX_ENCLAVE otherwise it is impossible to use sgx-ecdsa as verifier on the non-sgx platform. This is why we require carving up the enclave quote subsystem is so important. @YangLiang3 @Ruoyu-y 

But we can still mark sgx-ecdsa with ENCLAVE_QUOTE_OPTS_FLAGS_SGX_ENCLAVE because explicitly specifying -v sgx_ecdsa in commandline can enforce using sgx_ecdsa instance.